keyboards/sofle_pico/keymaps/default/keymap.c

```diff
@@ -0,0 +1,395 @@
+// Copyright 2024 Ryan Neff (@JellyTitan)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum sofle_layers {
+    /* _M_XYZ = Mac Os, _W_XYZ = Win/Linux */
+    _QWERTY,
+    _COLEMAK,
+    _LOWER,
+    _RAISE,
+    _ADJUST,
+};
+
+enum custom_keycodes { KC_QWERTY = SAFE_RANGE, KC_COLEMAK, KC_PRVWD, KC_NXTWD, KC_LSTRT, KC_LEND, KC_DLINE };
```

Comment:
As per the PR checklist, default keymaps are meant to be pristine, meaning no custom keycodes and whatnot. Unless these are "essential" to basic functionality of the keyboard, they should be removed.

Reply:
@waffle87 
I'm not attached to these guys, they came from the legacy `Sofle` keymaps. 

Two of those keycodes support the colemak/qwerty toggle, which i think is a neat feature - and I'd like to keep in there - but only if it's allowed - and if it's not too much trouble. 

The `KC_QWERTY` & `KC_COLEMAK` are used for setting `set_single_persistent_default_layer(layer)`. 

I dug through the docs and examples, and I couldn't find a way to use set_single_persistent_default_layer without a custom keycode. 

1) Is there a way to implement that feature without custom keycodes?
2) Are there formal, or even informal, guidelines on how simplified/prisitine the starter keymaps should be?

Reply:
Deleted custom keycodes, and removed related Colemak & toggle layers.